test/metabase/lib/drill_thru/distribution_test.cljc

```diff
@@ -18,7 +19,8 @@
       (if (and (= click :header)
                (not (#{:aggregation :breakout} column-kind))
                (not= column-type :pk)
-               (not (#{:type/Comment :type/Description} (:semantic-type (:column context)))))
+               (not (#{:type/Comment :type/Description} (:semantic-type (:column context))))
+               (not (lib.types.isa/structured? (:column context))))
         (is (canned/returned test-case context :drill-thru/distribution))
         (is (not (canned/returned test-case context :drill-thru/distribution)))))))
```

Comment:
Again that pattern, the test part of the `if` form is actually what `canned/returned` should produce.